src/molecules/Footer/footer.module.css

```diff
@@ -0,0 +1,12 @@
+footer {
```

Comment:
Please arrange properties in alphabetical order.

Reply:
Done.